src/components/pages/docs.js

```diff
@@ -180,16 +180,6 @@ class Docs extends React.Component {
     return <a href={urls.getDocUrl(doc.key, doc.slug)}>{doc.title}</a>;
   }
 
-  renderSlug(_value, doc) {
-    if (!doc.slug) {
-      const { t } = this.props;
-      return t('notAssigned');
```

Comment:
You can probably remove the translation too, right?

Reply:
I checked the other translations and forgot about this one. Thanks!

Reply:
Removed